dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -90,3 +94,6 @@ jwkEndpointUrl.desc=Specifies a JWK end point URL.
 useSystemPropertiesForHttpClientConnections=Use system properties for HTTP client connections
 useSystemPropertiesForHttpClientConnections.desc=Specifies whether to use Java system properties when the JWT Consumer creates HTTP client connections. Set this property to true if you want the connections to use the http* and javax* system properties.
 
+#Do not translate "amr"
+amrInclude=Authentication methods references in string array
+amrInclude.desc= Specifies Authentication Methods References so it can enforce step-up authentication and know how user is authenticated. Specify a comma separated list of valid amr claim values to be used.
```

Comment:
Please consider changing "how user is authenticated" to "how the user is authenticated".